tests/exposures/monkeypox_test.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+
+from conftest import FakeRandom
+
+from titan.exposures import MonkeyPox
+from titan.agent import Relationship
+
+
+@pytest.mark.unit
+def test_monkeypox_expose(make_model, make_agent):
+    model = make_model()
+    model.run_random = FakeRandom(0.0)  # always less than param
+    a = make_agent()
+    p = make_agent()
+    a.partners["Sex"] = set()
+    p.partners["Sex"] = set()
+    rel = Relationship(a, p, 10, bond_type="Sex")
+
+    MonkeyPox.expose(model, "sex", rel, 1)
+
+    assert a.monkeypox.active is False
+    assert p.monkeypox.active is False
+
+    a.monkeypox.active = True
+    a.monkeypox.time = model.time
+    p.monkeypox.active = True
+    a.monkeypox.time = model.time
+
+    # test nothing happens
+    MonkeyPox.expose(model, "sex", rel, 10)
+
+    assert a.monkeypox.active
+    assert p.monkeypox.active
+
+    p.monkeypox.active = False
+
+    # test conversion happens
+    MonkeyPox.expose(model, "sex", rel, 10)
+
+    assert a.monkeypox.active
+    assert p.monkeypox.active
```

Comment:
At this moment in time I had expected that `p.monkeypox.active` was false since in line 35 it was set to False. I guess i'll take a look at `expose` ...

Reply:
`expose` gives the agent the exposure!

Reply:
You know... that makes a lot of sense 🤦